app/models/user.rb

```diff
@@ -1,6 +1,7 @@
 class User < ActiveRecord::Base
 
-  validate :first_name, :last_name, :email, :novell_username, :webid
+  validate :email, :novell_username, :webid
+  validates_presence_of :first_name, :last_name
```

Comment:
So we should use here new rails validations syntax http://api.rubyonrails.org/classes/ActiveModel/Validations/ClassMethods.html#method-i-validates